src/index.html

```diff
@@ -1,19 +1,43 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="item-card" data-qa="card">
+      <img class="image-card" src="images/imac.jpeg" alt="Apple imac">
+      <div class="card-info">
+        <p class="card-header">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        <p class="card-code">Код товара: 195434</p>
+        <div class="card-rate">
+          <div class="card-sters">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star.svg" class="star" alt="star-active" width="13px" height="15px">
+          </div>
+          <p class="card-reviews">Отзывов: 5</p>
+        </div>
+        <div class="card-price">
+          <p class="price-text">Цена:</p>
+          <p class="price-number">69 999 грн</p>
+        </div>
+        <button type="button" class="card-button">
+          <a class="buttom-buy" data-qa="hover">Купить</a>
```

Comment:
Tag `<a>` should have `href` always. I meant remove script if not using, but wasn't right.